scout/server/blueprints/variant/templates/variant/cancer-variant.html

```diff
@@ -0,0 +1,363 @@
+{% endblock %}
+
+{% block css %}
+{{ super() }}
+<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-select@1.13.9/dist/css/bootstrap-select.min.css">
+{% endblock %}
+
+{% block top_nav %}
+  {{ super() }}
+  <li class="nav-item">
+    <a class="nav-link text-nowrap" href="{{ url_for('overview.cases', institute_id=institute._id) }}">
+      {{ institute.display_name }}
+    </a>
+  </li>
+  <li class="nav-item">
+    <a class="nav-link text-nowrap" href="{{ url_for('cases.case', institute_id=institute._id, case_name=case.display_name) }}">
+      {{ case.display_name }}
+    </a>
+  </li>
+  <li class="nav-item">
+    {% if cancer %}
+      <a class="nav-link text-nowrap" href="{{ url_for('variants.cancer_variants', institute_id=institute._id, case_name=case.display_name, variant_type=variant.variant_type, gene_panels=case.panels|selectattr('is_default')|map(attribute='panel_name')|list) }}">
+        {{ variant.variant_type|capitalize }} cancer variants
+      </a>
+    {% elif str %}
+      <a class="nav-link text-nowrap" href="{{ url_for('variants.str_variants', institute_id=institute._id, case_name=case.display_name, variant_type=variant.variant_type) }}">
+        {{ variant.variant_type|capitalize }} STR variants
+      </a>
+    {% else %}
+      <a class="nav-link text-nowrap" href="{{ url_for('variants.variants', institute_id=institute._id, case_name=case.display_name, variant_type=variant.variant_type, gene_panels=case.panels|selectattr('is_default')|map(attribute='panel_name')|list) }}">
+        {{ variant.variant_type|capitalize }} SNV and INDELs
+      </a>
+    {% endif %}
+  </li>
+  <li class="nav-item active">
+    <p class="navbar-text text-nowrap">{{ variant.display_name|truncate(20, True) }}</p>
+  </li>
+{% endblock %}
+
+{% block top_nav_right %}
+  {% if config['MAIL_USERNAME'] %}
+    {# Email setting must be setup #}
+    <li class="nav-item text-nowrap">{{ sanger_button(variant) }}</li>
+  {% endif %}
+  <li class="nav-item">{{ pin_button(variant, case, institute) }}</li>
+  {{ super() }}
+{% endblock %}
+
+{% block content_main %}
+  <div class="container-float">
+    {% if variant.missing_data %}
+    <div class="row">
+      <div class="alert alert-warning">Showing only first 30 genes!</div>
+    </div>
+    {% endif %}
+
+    {{ matching_variants(managed_variant) }}
+    <div class="card-columns">
+      {{ panel_classify(variant, institute, case, ACMG_OPTIONS, manual_rank_options, cancer_tier_options, dismiss_variant_options, mosaic_variant_options) }}
+      {{ panel_summary() }}
+      {{ frequencies(variant) }}
+
+      {% if variant.disease_associated_transcripts %}
+        <div class="col-sm-5">{{ disease_associated(variant) }}</div>
+      {% endif %}
+
+      {% if config['LOQUSDB_SETTINGS'] %}
+        {{ observations_panel(variant, observations, case) }}
+      {% endif %}
+
+      {{ panel_quality() }}
```

Comment:
Still checking the first variant of the demo case (TP53). Quality panel contains the mappability one, even if the latter should be separated (it's on line 156)

![image](https://user-images.githubusercontent.com/28093618/115832598-5c9c1980-a413-11eb-9f94-23759a3600be.png)